### PR TITLE
replace fmt.Printx; os.Exit(1) with log.Fatalx

### PR DESCRIPTION
**Pull request by briskt**

### CODE REVIEW

File: cmd/ecsReplaceInstances.go [hunk 1]
```diff
@@ -30,13 +31,11 @@ var replaceInstancesCmd = &cobra.Command{
...
 	Long:  ``,
 	Run: func(cmd *cobra.Command, args []string) {
-
 		initAwsSess()
 
 		asgName := lib.GetAsgNameForEcsCluster(AwsSess, cluster)
 		if asgName == "" {
-			fmt.Println("Unable to find ASG name for ECS cluster ", cluster)
-			os.Exit(1)
+			log.Fatalln("Unable to find ASG name for ECS cluster ", cluster)
```
Comment (jason-jackson) [inline cmd/ecsReplaceInstances.go:38]: *ln() should add a space between each parameter if I recall correctly. Can be changed multiple places.

```suggestion
			log.Fatalln("Unable to find ASG name for ECS cluster", cluster)
```

File: cmd/ecsRightSizeCluster.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ This function may scale a cluster up or down depending on services.`,
 		initAwsSess()
 		err := lib.RightSizeAsgForEcsCluster(AwsSess, cluster, atLeastServiceDesiredCount)
 		if err != nil {
-			os.Exit(1)
+			log.Fatalln(err.Error())
```
Comment (jason-jackson) [inline cmd/ecsRightSizeCluster.go:41]: I don't think you need the `.Error()`. Can be changed in multiple places too.

```suggestion
			log.Fatalln(err)
```

File: lib/asg.go [hunk 3]
```diff
@@ -144,8 +136,7 @@ func GetInstanceTypeForAsg(awsSess *session.Session, asgName string) string {
 		return GetInstanceTypeFromLaunchTemplate(awsSess, *asg.LaunchTemplate.LaunchTemplateName)
 	}
 
-	fmt.Println("Unable to determine the ASG instance type. No Launch Template nor Launch Configuration is defined.")
-	os.Exit(1)
+	log.Fatalln("Unable to determine the ASG instance type. No Launch Template nor Launch Configuration is defined.")
 	return ""
```
Comment (jason-jackson) [inline lib/asg.go:140]: I would think IDE would complain about unreachable code here. (Just a comment, no need to change)

File: lib/ecs.go [hunk 4]
```diff
@@ -145,8 +140,7 @@ func GetMemoryCpuNeededForEcsServices(awsSess *session.Session, ecsServices []*e
 			TaskDefinition: service.TaskDefinition,
 		})
 		if err != nil {
-			fmt.Printf("Unable to describe task definition %s\n", *service.TaskDefinition)
-			os.Exit(1)
+			log.Fatalf("Unable to describe task definition %s\n", *service.TaskDefinition)
```
Comment (jason-jackson) [inline lib/ecs.go:143]: Could do Fatalln like most of the others. (Not necessary)

**Review by jason-jackson — APPROVED** [review]
Looks good. Some minor considerations.

File: lib/asg.go (hunk 3 above)
Comment (briskt) [reply lib/asg.go:140]: Actually, Go requires it. Strange, but true.

**Pull request merged**